Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 
+RUN echo $PWD
+RUN ls
+RUN mkdir akka
+COPY . ./akka
+RUN ls
+WORKDIR /akka/src/core/Akka.Streams.Tests
+CMD ["/bin/bash", "-c", "dotnet restore Akka.Streams.Tests.csproj && dotnet test Akka.Streams.Tests.csproj --logger trx; exit 0"]
```

Comment:
Only tested with a single project but we can run all akka tests this way:
```
WORKDIR /akka/src
CMD ["/bin/bash", "-c", "dotnet restore && dotnet test Akka.sln --logger trx; exit 0"] 
```

Reply:
I'd probably do the following here:

* Make the Docker entry point programmable
* Add some comments explaining what this is for and when / why you would use it
* Documentation on how to run various types of tests through the entry point

Reply:
Okay